libs/net/dal/Migrations/20230921212403_1.0.69.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using TNO.DAL;
+
+#nullable disable
+
+namespace TNO.DAL.Migrations
+{
+    /// <inheritdoc />
+    public partial class _1069 : SeedMigration
```

Comment:
You did not correctly update your migration to include your latest configuration changes.  Make sure you rollback correctly.  Ask if you forget.

Reply:
20th time the charm, updated again